src/app/clusters/icd-management-server/icd-management-server.cpp

```diff
@@ -304,18 +322,19 @@ void ICDManagementServer::TriggerICDMTableUpdatedEvent()
     ICDNotifier::GetInstance().BroadcastICDManagementEvent(ICDListener::ICDManagementEvents::kTableUpdated);
 }
 
-CHIP_ERROR ICDManagementServer::CheckAdmin(CommandHandler * commandObj, const ConcreteCommandPath & commandPath, bool & isAdmin)
+CHIP_ERROR ICDManagementServer::CheckAdmin(CommandHandler * commandObj, const ConcreteCommandPath & commandPath,
```

Comment:
does this method use `this` in any way? If not could we remove it from the class and place it in an anonymous namespace?

Reply:
No, function doesn't need `this`. We could move it to an anonymous namespace or make it static within the class. Don't have a preference.

Reply:
I think it has no reason to appear in the header then, so we may as well use anon namespace for that reason (smaller header is smaller api for anyone reading the code)

Reply:
done